KubeArmor/config/config.go

```diff
@@ -144,9 +150,11 @@ func LoadConfig() error {
 		GlobalCfg.Policy = false
 		GlobalCfg.HostPolicy = true
 	}
+	GlobalCfg.K8sEnv = viper.GetBool(ConfigK8sEnv)
+	kl.IsK8SEnv = viper.GetBool(ConfigK8sEnv)
```

Comment:
Please store the IsK8sEnv in GlobalCfg and then use GlobalCfg in kl module.

Reply:
It is not possible to include config in common as it causes circular dependency. Hence this global variable.